js/arrangeBooks.js

```diff
@@ -1,53 +1,56 @@
+    this.bookShelf = JSON.parse(localStorage.getItem('bookShelf')) || [];
+    this.init();
   }
-}
-
-function displayBook(book) {
-  const singleBook = document.createElement('li');
-  singleBook.setAttribute('id', `${book.title}-${book.author}`);
-
-  const bookPrint = document.createElement('span');
-  bookPrint.innerHTML = `${book.title}<br>${book.author}<br>`;
-  const removeButton = document.createElement('button');
-  removeButton.innerText = 'Leave team';
-  removeButton.addEventListener('click', () => removeBook(book.title, book.author));
 
-  singleBook.appendChild(bookPrint);
-  singleBook.appendChild(removeButton);
-  bookList.appendChild(singleBook);
-}
-
-function addBook(title, author) {
-  const newBook = {
-    title,
-    author,
-  };
+  init() {
+    this.bookShelf.forEach((book) => this.displayBook(book));
+    this.submitButton.addEventListener('click', () => {
+      const title = document.getElementById('bookTitle').value;
+      const author = document.getElementById('bookAuthor').value;
+      this.addBook(title, author);
+    });
+  }
 
-  bookShelf.push(newBook);
+  removeBook(title, author) {
+    this.bookShelf = this.bookShelf.filter(
+      (book) => book.title !== title || book.author !== author,
+    );
+    localStorage.setItem('bookShelf', JSON.stringify(this.bookShelf));
+    const singleBook = document.getElementById(`${title}-${author}`);
+    if (singleBook) {
+      this.bookList.removeChild(singleBook);
+    }
+  }
 
-  localStorage.setItem('bookShelf', JSON.stringify(bookShelf));
-  document.getElementById('bookTitle').value = '';
-  document.getElementById('bookAuthor').value = '';
+  displayBook(book) {
+    const singleBook = document.createElement('li');
+    singleBook.setAttribute('id', `${book.title}-${book.author}`);
+    const bookPrint = document.createElement('span');
+    bookPrint.innerHTML = `${book.title} by ${book.author}<br>`;
+    const removeButton = document.createElement('button');
+    removeButton.innerText = 'Remove';
+    removeButton.addEventListener('click', () => this.removeBook(book.title, book.author));
+    singleBook.appendChild(bookPrint);
+    singleBook.appendChild(removeButton);
+    this.bookList.appendChild(singleBook);
+  }
 
-  displayBook(newBook);
+  addBook(title, author) {
+    const newBook = {
+      title,
+      author,
+    };
+    this.bookShelf.push(newBook);
+    localStorage.setItem('bookShelf', JSON.stringify(this.bookShelf));
+    document.getElementById('bookTitle').value = '';
+    document.getElementById('bookAuthor').value = '';
+    this.displayBook(newBook);
+  }
 }
 
-bookShelf.forEach((book) => displayBook(book));
-
-submitButton.addEventListener('click', () => {
-  const title = document.getElementById('bookTitle').value;
-  const author = document.getElementById('bookAuthor').value;
-  addBook(title, author);
-});
\ No newline at end of file
+const bookShelf = new BookShelf();
+console.log(bookShelf);
```

Comment:
- Please remove all `console.log` statement from your code, this will ensure that your code is readable and easy to maintain 👍

Reply:
Hi Mckent,

I hope you're doing well. I'm writing to ask for some additional guidance on how to fix the "Missing script: start" error that I encountered when running the "npm run" command.

 I'm not sure what specifically needs to be added or changed in the file to fix this error.

Could you please provide me with more detailed guidance on how to fix this error? Any additional information you can provide would be greatly appreciated.

Thank you for your help!